kinesis_firehose/output.tf

```diff
@@ -0,0 +1,12 @@
+output "firehose_delivery_stream_arn" {
```

Comment:
What is `kinesis_firehose/minimum.out` binary file?

Reply:
Its the .out file that we use. Got checked in my mistake. Will be reverting it out

Reply:
still there